src/deluge/gui/menu_item/value_scaling.cpp

```diff
@@ -16,16 +16,13 @@ int32_t computeCurrentValueForPan(int32_t value) {
 	return ((int64_t)value * (kMaxMenuRelativeValue * 2) + 2147483648) >> 32;
 }
 
-int32_t computeCurrentValueForArpMidiCvGate(int32_t value) {
-	return computeCurrentValueForStandardMenuItem(value);
-}
-
 int32_t computeCurrentValueForArpMidiCvRatchetsOrRhythm(uint32_t value) {
 	return ((int64_t)value * kMaxMenuValue + 2147483648) >> 32;
 }
 
-int32_t computeCurrentValueForArpMidiCvRate(int32_t value) {
-	return computeCurrentValueForStandardMenuItem(value);
+int32_t computeFinalValueForSemiStandardMenuItem(int32_t value) {
```

Comment:
For clarity - computeFinalValueWithoutRoundToOne?

That explains what it does better than just being semi standard

Reply:
Done, though that naming scheme has the minor issue of not being symmetrical:

`computeCurrentValueWithoutRoundingToOne()`
`computeFinalValueWithoutRoundingToOne()`

Unless the reader has fully internalized Q31 usage it's really not obvious what rounding it's speaking about. 😅 ...but since I don't expect these functions will live long after the complete value scaling scheme has been extracted for examination, I don't think that's an issue. :)